### PR TITLE
M51025: Fixing typo customer/custom

### PR DESCRIPTION
**Pull request by v-maudel**
Translator has reported possible source content issue.

### CODE REVIEW

**Comment by v-maudel** [comment]
Hi @KathleenDollard, can you please help us by merging this PR?
Regards

File: docs/visual-basic/programming-guide/language-features/data-types/tuples.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ A Visual Basic tuple is a value type that is an instance of one of the a **Syste
 
 If custom members, property and field validation, or immutability are important, you should use the Visual Basic [Structure](../../../language-reference/statements/structure-statement.md) statement to define a custom value type.
 
-A Visual Basic tuple does inherit the members of its **ValueTuple** type. In addition to its fields, these include the following methods:
+A Visual Basic tuple does inherit the members of its **ValueTuple** type. In addition to its fields, these include the following members:
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/language-features/data-types/tuples.md:79]: These are all methods so not sure why we need to change it here.

But the change from customer to custom is correct.

Comment (BillWagner) [reply docs/visual-basic/programming-guide/language-features/data-types/tuples.md:79]: I agree with @mairaw here. Can you revert this change?

**Review by BillWagner — DISMISSED** [review]
Thanks for making this @v-maudel 

I agree with Maira that the one change should be reverted. Once that's done, I'll :shipit:

**Comment by CeciAc** [comment]
Hi all, can you please merge this pr?


@mairaw @BillWagner

**Review by adegeo — APPROVED** [review]

**Comment by adegeo** [comment]
Approving. I edited the source to change the value according to the reviews.

**Pull request merged**